src/devices/bresser_3ch.c

```diff
@@ -51,7 +51,7 @@ static int bresser_3ch_callback(bitbuffer_t *bitbuffer) {
     b[4] = ~b[4];
 
     if (((b[0] + b[1] + b[2] + b[3] - b[4]) & 0xFF) != 0) {
-        if (debug_output) {
+        if (debug_output >= 1) {
```

Comment:
Same comment as above, not overly verbose, so should require a higher debug level.